source/core/types/ut_test.tpb

```diff
@@ -36,19 +36,17 @@ create or replace type body ut_test as
 
     ut_utils.debug_log('ut_test.execute');
 
+    a_listener.fire_before_event(ut_utils.gc_test,self);
+    self.start_time := current_timestamp;
+
     if self.get_ignore_flag() then
       self.result := ut_utils.tr_ignore;
       ut_utils.debug_log('ut_test.execute - ignored');
-      self.start_time := current_timestamp;
+      self.results_count := ut_results_counter(self.result);
       self.end_time := current_timestamp;
```

Comment:
should this become `self.end_time := self.start_time`  or do you really want to time those 3 lines?